analyzers/build/utils.go

```diff
@@ -0,0 +1,14 @@
+package build
+
+import (
+	"crypto/rand"
+	"fmt"
+)
+
+func generateRandomTag(length int) string {
```

Comment:
This should be wherever `docker` package is called. Rename this to `generateImageVersion`.